doc/aliases/add_toggle.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+/*
+  Creates a toggleable div that includes
+  the text/code associated to a button.
+*/
+var $getDiv = $('div.newInnerHTML').last();
+var buttonName = $getDiv.html();
+var label = getLabelName(buttonName.trim());
```

Comment:
Yes, I see. And we can also have predefined aliases for often used blocks for convenience. Thus I think that additional processing can be avoided.